views/index.html.twig

```diff
@@ -1,183 +1,118 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="robots" content="noindex,nofollow" />
-    <title>{{ name }} Composer Repository</title>
+
+    <title>{{ name }} Composer repository</title>
+
     <style type="text/css">
-    {% include 'styles.css' %}
+        {% include 'assets/bootstrap.min.css' %}
```

Comment:
Fair enough. I think it should be reasonably easy to split this into 2 pull requests.